dashboard/app/controllers/potential_teachers_controller.rb

```diff
@@ -9,7 +9,7 @@ def create
       begin
         @potential_teacher.save!
         render json: {message: 'Potential teacher created successfully'}, status: :created
-      rescue ActiveRecord::RecordInvalid => exception
+      rescue => exception
```

Comment:
I'd personally remove this? I'm not sure what this adds, other than adding "ERROR: " to the beginning of the message. I _think_ (but am not sure) that this would return a 500 either way.

Reply:
Alternatively, you could keep this and consider printing `@potential_teacher.errors.full_messages` ([documentation](https://api.rubyonrails.org/v6.1.4/classes/ActiveModel/Errors.html#method-i-full_messages)) to add more context to the Honeybadger error that (I think) would be raised in this case.